printer_mods/hartk1213/Voron2.4_GE5C/readme.md

```diff
@@ -0,0 +1,44 @@
+# Voron 2.4 GE5C Z joint  
+
+
+##
+
+### BOM
+Screws			
+Size|Qty|
```

Comment:
This markdown table is broken, it displays as a single line.  See https://www.markdownguide.org/extended-syntax/#tables for how to format the table.

Reply:
![image](https://user-images.githubusercontent.com/12398294/136728361-203829a8-e3bf-4320-bc3d-695a4198afa0.png)
its weird mine shows the table as working

Reply:
It's a strange GitHub Markdown syntax bypass.  If you add a vertical line (|) at the beginning of every line it should fix it.

Reply:
Fixed